Index out of range in BindsToAllNetworkInterfaces

```
$ gas main.go
[gas] 2017/11/23 10:03:58 Processing "~/go-workspace/src/github.com/repro/gas-error/main.go"...
panic: runtime error: index out of range

goroutine 1 [running]:
github.com/GoASTScanner/gas/rules.(*BindsToAllNetworkInterfaces).Match(0xc42007b8c0, 0x14a1f60, 0xc4200528c0, 0xc4200523c0, 0x0, 0x0, 0x0)
	~/go-workspace/src/github.com/GoASTScanner/gas/rules/bind.go:33 +0x176
github.com/GoASTScanner/gas/core.(*Analyzer).Visit(0xc420052380, 0x14a1f60, 0xc4200528c0, 0x14a0860, 0xc420052380)
	~/go-workspace/src/github.com/GoASTScanner/gas/core/analyzer.go:220 +0x1b1
go/ast.Walk(0x14a0860, 0xc420052380, 0x14a1f60, 0xc4200528c0)
	/usr/local/opt/go/libexec/src/go/ast/walk.go:52 +0x66
go/ast.walkExprList(0x14a0860, 0xc420052380, 0xc420048a40, 0x1, 0x1)
	/usr/local/opt/go/libexec/src/go/ast/walk.go:26 +0x81
go/ast.Walk(0x14a0860, 0xc420052380, 0x14a1d60, 0xc420052900)
	/usr/local/opt/go/libexec/src/go/ast/walk.go:207 +0x2122
go/ast.walkStmtList(0x14a0860, 0xc420052380, 0xc42000aa20, 0x2, 0x2)
	/usr/local/opt/go/libexec/src/go/ast/walk.go:32 +0x81
go/ast.Walk(0x14a0860, 0xc420052380, 0x14a1ee0, 0xc42007bad0)
	/usr/local/opt/go/libexec/src/go/ast/walk.go:224 +0x1b7a
go/ast.Walk(0x14a0860, 0xc420052380, 0x14a2360, 0xc42007bb00)
	/usr/local/opt/go/libexec/src/go/ast/walk.go:344 +0xd83
go/ast.walkDeclList(0x14a0860, 0xc420052380, 0xc42000aa60, 0x2, 0x2)
	/usr/local/opt/go/libexec/src/go/ast/walk.go:38 +0x81
go/ast.Walk(0x14a0860, 0xc420052380, 0x14a22e0, 0xc4200a8380)
	/usr/local/opt/go/libexec/src/go/ast/walk.go:353 +0x2672
github.com/GoASTScanner/gas/core.(*Analyzer).process(0xc420052380, 0xc420016550, 0x49, 0x0, 0x0, 0x2, 0xc4200740c0)
	~/go-workspace/src/github.com/GoASTScanner/gas/core/analyzer.go:138 +0x78a
github.com/GoASTScanner/gas/core.(*Analyzer).Process(0xc420052380, 0xc420016550, 0x49, 0xc42006fef0, 0x1)
	~/go-workspace/src/github.com/GoASTScanner/gas/core/analyzer.go:160 +0x55
main.main()
	~/go-workspace/src/github.com/GoASTScanner/gas/main.go:199 +0x7b6
```

This is error is easy reproducible with the following sample code in the main.go:

```
package main

import (
	"github.com/eawsy/aws-lambda-go-net/service/lambda/runtime/net"
)

func main() {
	ln := net.Listen()
	ln.Accept()

}
```

--- a/call_list.go
+++ b/call_list.go
@@ -61,14 +61,18 @@ func (c CallList) ContainsCallExpr(n ast.Node, ctx *Context) *ast.CallExpr {
 		return nil
 	}
 
-	// Try direct resolution
-	if c.Contains(selector, ident) {
-		return n.(*ast.CallExpr)
-	}
-
-	// Also support explicit path
+	// Use only explicit path to reduce conflicts
 	if path, ok := GetImportPath(selector, ctx); ok && c.Contains(path, ident) {
 		return n.(*ast.CallExpr)
 	}
+
+	/*
+		// Try direct resolution
+		if c.Contains(selector, ident) {
+			log.Printf("c.Contains == true, %s, %s.", selector, ident)
+			return n.(*ast.CallExpr)
+		}
+	*/
+
 	return nil
 }

--- a/call_list_test.go
+++ b/call_list_test.go
@@ -66,7 +66,7 @@ var _ = Describe("call list", func() {
 		ctx := pkg.CreateContext("md5.go")
 
 		// Search for md5.New()
-		calls.Add("md5", "New")
+		calls.Add("crypto/md5", "New")
 
 		// Stub out visitor and count number of matched call expr
 		matched := 0

--- a/rules/bind.go
+++ b/rules/bind.go
@@ -46,7 +46,7 @@ func (r *bindsToAllNetworkInterfaces) Match(n ast.Node, c *gas.Context) (*gas.Is
 func NewBindsToAllNetworkInterfaces(conf gas.Config) (gas.Rule, []ast.Node) {
 	calls := gas.NewCallList()
 	calls.Add("net", "Listen")
-	calls.Add("tls", "Listen")
+	calls.Add("crypto/tls", "Listen")
 	return &bindsToAllNetworkInterfaces{
 		calls:   calls,
 		pattern: regexp.MustCompile(`^(0.0.0.0|:).*$`),

--- a/rules/rsa.go
+++ b/rules/rsa.go
@@ -39,7 +39,7 @@ func (w *weakKeyStrength) Match(n ast.Node, c *gas.Context) (*gas.Issue, error) 
 // NewWeakKeyStrength builds a rule that detects RSA keys < 2048 bits
 func NewWeakKeyStrength(conf gas.Config) (gas.Rule, []ast.Node) {
 	calls := gas.NewCallList()
-	calls.Add("rsa", "GenerateKey")
+	calls.Add("crypto/rsa", "GenerateKey")
 	bits := 2048
 	return &weakKeyStrength{
 		calls: calls,

--- a/rules/subproc.go
+++ b/rules/subproc.go
@@ -52,7 +52,7 @@ func (r *subprocess) Match(n ast.Node, c *gas.Context) (*gas.Issue, error) {
 // NewSubproc detects cases where we are forking out to an external process
 func NewSubproc(conf gas.Config) (gas.Rule, []ast.Node) {
 	rule := &subprocess{gas.NewCallList()}
-	rule.Add("exec", "Command")
+	rule.Add("os/exec", "Command")
 	rule.Add("syscall", "Exec")
 	return rule, []ast.Node{(*ast.CallExpr)(nil)}
 }

--- a/rules/tempfiles.go
+++ b/rules/tempfiles.go
@@ -39,7 +39,7 @@ func (t *badTempFile) Match(n ast.Node, c *gas.Context) (gi *gas.Issue, err erro
 // NewBadTempFile detects direct writes to predictable path in temporary directory
 func NewBadTempFile(conf gas.Config) (gas.Rule, []ast.Node) {
 	calls := gas.NewCallList()
-	calls.Add("ioutil", "WriteFile")
+	calls.Add("io/ioutil", "WriteFile")
 	calls.Add("os", "Create")
 	return &badTempFile{
 		calls: calls,

--- a/rules/templates.go
+++ b/rules/templates.go
@@ -41,10 +41,10 @@ func (t *templateCheck) Match(n ast.Node, c *gas.Context) (*gas.Issue, error) {
 func NewTemplateCheck(conf gas.Config) (gas.Rule, []ast.Node) {
 
 	calls := gas.NewCallList()
-	calls.Add("template", "HTML")
-	calls.Add("template", "HTMLAttr")
-	calls.Add("template", "JS")
-	calls.Add("template", "URL")
+	calls.Add("html/template", "HTML")
+	calls.Add("html/template", "HTMLAttr")
+	calls.Add("html/template", "JS")
+	calls.Add("html/template", "URL")
 	return &templateCheck{
 		calls: calls,
 		MetaData: gas.MetaData{

